lib/00_core.sh

```diff
@@ -119,7 +119,7 @@ _PACMAN_detect() {
   [ -x "/usr/bin/apt-get" ] && _PACMAN="dpkg" && return
   [ -x "/data/data/com.termux/files/usr/bin/apt-get" ] && _PACMAN="dpkg" && return
   [ -x "/usr/bin/cave" ] && _PACMAN="cave" && return
-  [ -x "/usr/bin/dnf" ] && _PACMAN="dnf" && return
+  [[ -x "/usr/bin/dnf" || -L "/usr/bin/dnf" ]] && _PACMAN="dnf" && return
```

Comment:
I've added this line because `fedora:34` docker image is not exposing a executable in `/usr/bin/dnf`, it is a symlink, although, I'm not sure about this change.

Reply:
@mondeja You may want to use `[ -x "/usr/bin/dnf" ] || [ -L "/usr/bin/dnf" ]`, to get `make POSIX` passed.

That said, the `-x` would return true if the target is executable. So I think the existing test just works fine.

Reply:
It seems that [in the CI works without this change](https://github.com/mondeja/pacapt/runs/3006693211), so let it as is

Reply:
I'd suggest it's how `symbolic` link and the test `[ -x foo ]` are supposed to work.